### ISSUE TITLE
How to show overlap points in scatter plot?

### ISSUE THREAD

**Issue by vologdin** [plotly/plotly.py#2531]
When two points have same [x,y] values, the scatter plot only show one point. Is there a way to display all points and their hover info even when there are overlap?

I can not use `fig.update_layout (hovermode = 'x')` because I have so many points on the same X axis

import plotly.graph_objects as go
fig = go.Figure()
fig.add_trace(go.Scatter(
    x=[0, 1, 2,1,2],
    y=[3, 3, 3,3,3],
    mode="markers",
 text = ['m1', 'm2', 'm3','m4','m5']
)
             )
fig.show()

![image](https://user-images.githubusercontent.com/31883685/83642391-75130380-a5b7-11ea-8c5a-c4df6ade4249.png)

**Comment by nicholas-esterer** [comment]
I'd have to look deeper to see if it's possible for a single trace but if you partition your points into 2 traces you can make it work:
```python
import plotly.graph_objects as go
fig = go.Figure()
fig.add_trace(go.Scatter(
x=[0, 1, 2],
y=[3, 3, 3],
mode="markers",
text = ['m1', 'm2', 'm3'],
marker=dict(color='blue')
)
)
fig.add_trace(go.Scatter(
x=[1,2],
y=[3,3],
mode="markers",
text = ['m4','m5'],
marker=dict(color='blue')
)
)
fig.update_layout(hovermode='x')
fig.show()
```
![image](https://user-images.githubusercontent.com/64649603/83650977-f900c780-a586-11ea-994c-071684ab21ef.png)

**Comment by nicolaskruchten** [comment]
Indeed: our hover labels today only ever display once per trace, on the "top most" point.
